### PR TITLE
Add config of TiFlash coprocessor thread pool

### PR DESCRIPTION
**Pull request by LittleFall**
### What is changed, added or deleted? (Required)

Add config of TiFlash coprocessor thread pool.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tics/pull/1280/

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
> /label version-specific-changes-required
> 
> When it is cherry-picked to branch 5.0, should remove description `4.0.11 引入，`.

If so, do we really need to cherry-pick this to release-5.0?

**Comment by LittleFall** [comment]
> > /label version-specific-changes-required
> > When it is cherry-picked to branch 5.0, should remove description `4.0.11 引入，`.
> 
> If so, do we really need to cherry-pick this to release-5.0?

Yes, we need it, I mean:

- For 5.X, every version has this feature.
- For 4.X, this feature is imported in 4.0.11.

Or do you have any suggestions for more clear description? Thanks!

**Review by zanmato1984 — APPROVED** [review]
LGTM

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -117,6 +117,10 @@ minmax_index_cache_size = 5368709120
     max_memory_usage = 0
     ## 所有查询过程中，对中间数据的内存限制，单位为 byte，默认为 0，表示不限制
     max_memory_usage_for_all_queries = 0
+    ## 4.0.11 引入，表示 TiFlash Coprocessor 最多同时执行的 cop 请求数量，更多的请求会排队等待，设为 0 或不设置则使用默认值：物理核数的两倍
```
Comment (TomShawn) [inline tiflash/tiflash-configuration.md:120]:
```suggestion
    ## 从 v4.0.11 引入，表示 TiFlash Coprocessor 最多同时执行的 cop 请求数量。如果请求数量超过了该配置指定的值，多出的请求会排队等待。如果设为 0 或不设置，则使用默认值，即物理核数的两倍。
```

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -117,6 +117,10 @@ minmax_index_cache_size = 5368709120
...
+    cop_pool_size = 0
+    ## 4.0.11 引入，表示 TiFlash Coprocessor 最多同时执行的 batch 请求数量，更多的请求会排队等待，设为 0 或不设置则使用默认值：物理核数的两倍
```
Comment (TomShawn) [inline tiflash/tiflash-configuration.md:122]:
```suggestion
    ## 从 v4.0.11 引入，表示 TiFlash Coprocessor 最多同时执行的 batch 请求数量。如果请求数量超过了该配置指定的值，多出的请求会排队等待。如果设为 0 或不设置，则使用默认值，即物理核数的两倍。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**